frontend/src/pages/Homepage.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import {BrowserRouter as Router, Route} from "react-router-dom";
+import {Link} from 'react-router-dom';
+import {Grid, Image, Header, Container, Button, Divider, Segment} from 'semantic-ui-react'
+import '../styles/Homepage.css'
+
+function Homepage() {
```

Comment:
Code should be taken out of JSX and changes in page should be handled by code like this: https://tylermcginnis.com/react-router-programmatically-navigate/